spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/RemoteTokenServices.java

```diff
@@ -109,6 +109,13 @@ public OAuth2Authentication loadAuthentication(String accessToken) throws Authen
 		headers.set("Authorization", getAuthorizationHeader(clientId, clientSecret));
 		Map<String, Object> map = postForMap(checkTokenEndpointUrl, formData, headers);
 
+		if (map == null) {
+			if (logger.isDebugEnabled()) {
+				logger.debug("check_token returned empty: " + accessToken);
```

Comment:
We should not log credentials (accessToken) as this leaks data.

Reply:
> We should not log credentials (accessToken) as this leaks data.

ok.  I have removed

Reply:
Thanks! I assigned to @jgrandja who will help guide you with getting this merged